docs/spec.json

```diff
@@ -141,7 +141,13 @@
           "description": "Print a list of all generic dimensions in the current app"
         },
         "properties": {
-          "description": "Print the properties of the generic dimension"
+          "description": "Print the properties of the generic dimension",
+          "flags": {
+            "minimum": {
```

Comment:
Not sure if `minimum` is the correctl flag name for this, since the property tree returned is quite verbose even without the client properties for e.g. objects. Perhaps `--engine-only`?